### ISSUE TITLE
Enhancement: tool to validate exports against branch

### ISSUE THREAD

**Issue by vanderaj** [OWASP/ASVS#1300]
We need a tool that checks translations have all requirements, and highlight missing or extra requirements.

**Comment by elarlang** [comment]
Depends, what kind problem we need to solve with this tool.

From merging some translation from PR to the main repo, I have it covered. Since v4.0.3, each line of the translation is checked and compared against (a bit fixed) translation branch. So in process, we don't have this problem.

If we want to provide this as a tool for translations, I really recommend, that we keep the original content in some other way than github markdown format. But this means we need to change current technical process entirely.

In short - I think we are ok now, we don't have huge amount of translations coming in and for serving later content from ASVS repo, those are validated.

**Comment by tghosth** [comment]
I think this has value for translations as even if the original MD is ok, often there are problems in the output exports caused by non english character peculiarities.

**Comment by elarlang** [comment]
I think we don't need extra tool as such (I'm the tool at the moment), but need to make clear translation process.

The main point is - it does not make sense to translate markdown, but it makes sense to create structured ASVS (or anything else) and with that we can provided piece-by-piece translation options.
